data/meson.build

```diff
@@ -0,0 +1,107 @@
+
+icon_sizes = ['16x16', '22x22', '24x24', '32x32', '48x48', '256x256', '512x512']
+
+# install bitmap icons
+foreach size : icon_sizes
+    install_data(
+        'icons/hicolor/' + size + '/apps/' + meson.project_name() + '.png',
+        install_dir: 'share/icons/hicolor/' + size + '/apps/'
+    )
+endforeach
+
+# install scalable symbolic icon
+install_data(
+    'icons/hicolor/scalable/apps/com.gexperts.Tilix-symbolic.svg',
+    install_dir: 'share/icons/hicolor/scalable/apps/'
+)
+
+
+# Validate desktop file
+desktop_file_validate = find_program('desktop-file-validate', required:false)
+
+if desktop_file_validate.found()
+  test (
+    'Validate desktop file',
+    desktop_file_validate,
+    args: join_paths(meson.current_build_dir (), meson.project_name() + '.desktop')
+  )
+endif
+
+# Install the desktop file
+i18n.merge_file(
+    output:  meson.project_name() + '.desktop',
+    input: 'pkg/desktop/' + meson.project_name() + '.desktop.in',
+    po_dir: join_paths(meson.source_root(), 'po'),
+    type: 'desktop',
+    install: true,
+    install_dir: 'share/applications'
+)
+
+# Validate AppData file
+appstreamcli = find_program('appstream-util', required:false)
```

Comment:
Any reason for using `appstream-util` over the reference `appstreamcli` here?
If you prefer to use asutil, I would like to know the reason for that. In any case, please don't call it appstreamcli and also use its `validate-relax` method in the build process, otherwise compiling Tilix will be a pain if asutil is installed and more up-to-date than what Tilix was last tested with.

Reply:
@bilelmoussaoui That's not true, it supports `content_rating` since AppStream version 0.11.0 which was released in May 2017, more than a year ago.

Reply:
Well, we had that issue with FeedReader using the latest Fedora

Reply:
Can you point me at the issue? The only thing I'm aware of that was recently changed was empty content_attribute tags being allowed instead of setting every attribute to none explicitly.

Reply:
That's exactly what I was talking about

Reply:
Please report these things next time! Also keep in mind that an empty `content_rating` is not yet supported by all tools, so using it at the moment is still a bit risky (I actually only noticed this because of an error related to it).

Reply:
Reported that! Thanks for your work on appstream! :)